src/envoy/auth/jwt_test.cc

```diff
@@ -307,11 +307,23 @@ class JwtTestWithJwk : public testing::Test {
       "37BD2KXH3jN9_EBocxwcxhkPwSLtP3dgkfls_f5GoWCgmp-c5ycIskCDcIjxRnPjg";
 };
 
+#define TEST_SETUP(public_key, jwt, verified) \
```

Comment:
Use "Value Parameterized Tests".

See https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md

Reply:
Do you mean something like this?:
```
vector<tuple<string, string, bool> > testdata = {
  make_tuple(jwt1, pubkey1, verify_result1),
  make_tuple(jwt2, pubkey2, verify_result2),
  ...
};
INSTANTIATE_TEST_CASE_P(foobar, JwtTest, testing::ValuesIn(testdata));
...
```

Reply:
done